lib/Ed25519Signature2020.js

```diff
@@ -49,7 +51,6 @@ export class Ed25519Signature2020 extends LinkedDataSignature {
       contextUrl: SUITE_CONTEXT_URL,
       key, signer, verifier, proof, date, useNativeCanonize
     });
-    this.requiredKeyType = 'Ed25519VerificationKey2020';
```

Comment:
I think this `requiredKeyType` would be considered part of the public API here.  Therefore removing it would constitute a breaking change.  I'm not sure we want to make a breaking change for something this minor.  We also cannot change the data type from `String` to `Array` (to include the two valid key types) without breakage.  

While not strictly required, it would *also* be nice to maintain parity with other signature modules:
https://github.com/digitalbazaar/ed25519-signature-2018/blob/main/lib/Ed25519Signature2018.js#L48

I think leaving this line of code along with a comment along these lines is the best we can do at this point:
```
// some operations may be performed with Ed25519VerificationKey2018 so Ed25519VerificationKey2020
// is recommended, but not strictly required
```

Reply:
yeah, that makes sense. I'll add back the line and add comment to it, thank you.

Reply:
Addressed here b55745a